crates/biome_js_syntax/src/expr_ext.rs

```diff
@@ -512,6 +513,10 @@ impl JsNumberLiteralExpression {
     pub fn as_number(&self) -> Option<f64> {
         parse_js_number(self.value_token().unwrap().text())
     }
+
+    pub fn inner_string_text(&self) -> SyntaxResult<TokenText> {
+        Ok(inner_string_text(&self.value_token()?))
```

Comment:
I am not sure it makes sense of calling/defining `inner_string_text` here because it si intended to retrieve a text between quotes ("string text"). If you need to retrieve a `TokenText`, you can use at call site the following code: `number_expr.value_token()?.token_text()`

Reply:
You are right, we don't need quote handling for numbers. I remove it.

Reply:
In your case, you can even simply use `token.text_trimmed`.